apps/src/aichat/redux/aichatRedux.ts

```diff
@@ -173,6 +173,14 @@ const getNewMessageId = () => {
   return latestMessageId;
 };
 
+const RESET_MODEL_NOTIFICATION: ChatCompletionMessage = {
```

Comment:
@bencodeorg added a couple updates since your last review. @samantha-code confirmed she did want a notification message displayed when user starts over. Thanks!